ossdbtoolsservice/object_explorer/object_explorer_service.py

```diff
@@ -383,3 +386,7 @@ def _route_request(self, is_refresh: bool, session: ObjectExplorerSession, path:
         else:
             # Return the results of a previous request for the same path
             return session.cache[path]
+    
+    def _invalidate_cache(self, session: ObjectExplorerSession, path: str) -> None:
```

Comment:
Since we are not invalidating the whole cache here, you can rename it to something more precise like _remove_path_from_cache

Reply:
done